pkg/crc/machine/types/types.go

```diff
@@ -30,7 +30,8 @@ type StartConfig struct {
 	Preset crcpreset.Preset
 
 	// Shared dirs
-	EnableSharedDirs bool
+	EnableSharedDirs     bool
+	SMBSharedDirPassword string
```

Comment:
Can the password be set per directory share? Or is it always going to be the same? It's odd to have password there, and to guess username later on.

Reply:
its the same password always, when the shares are created access controls are based on windows users and groups 

one way to do this would be to create a new user and only allow that user to access, but then it appears in the login, there's some way to hide the user from the login screen by editing registry keys, but seem too hacky and haven't tried it yet https://www.windowscentral.com/how-hide-specific-user-accounts-sign-screen-windows-10

Reply:
>  It's odd to have password there, and to guess username later on.

yeah, the username should also be part of `StartConfig` tthen, but would prefer not to expose it as a config for the user to set (less things to configure is nicer imo)